nft-candy-machine/js/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@metaplex-foundation/mpl-candy-machine",
+  "version": "0.0.1",
+  "description": "MPL Candy Machine JavaScript API. This MPL package is for the current generation of candy machine which is 2.0.0",
+  "main": "dist/src/mpl-candy-machine.js",
+  "types": "dist/src/mpl-candy-machine.d.ts",
+  "scripts": {
+    "check:publish-ready": "yarn build && yarn test",
+    "preversion": "yarn check:publish-ready",
+    "postversion": "git add package.json && git commit -m \"chore: update $npm_package_name to v$npm_package_version\" && git tag $npm_package_name@$npm_package_version",
+    "prepublish": "yarn check:publish-ready",
+    "postpublish": "git push origin && git push origin --tags",
+    "build:docs": "typedoc",
+    "build": "rimraf dist && tsc -p tsconfig.json",
+    "test": "echo mpl-candy-machine tests coming up ...",
+    "api:gen": "node scripts/api-gen-ts.js && prettier --write ./src/**/*.ts",
+    "lint": "eslint \"{src,test}/**/*.ts\" --format stylish",
+    "fix:lint": "yarn lint --fix",
+    "prettier": "prettier \"{src,test}/**/*.ts\" --check",
+    "fix:prettier": "prettier --write src/",
+    "fix": "yarn fix:lint && yarn fix:prettier"
+  },
+  "files": [
+    "/dist"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "keywords": [
+    "nft",
+    "metaplex",
+    "solana",
+    "blockchain"
+  ],
+  "homepage": "https://metaplex.com",
+  "repository": "https://github.com/metaplex-foundation/metaplex-program-library.git",
+  "author": "Metaplex Maintainers <contact@metaplex.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@metaplex-foundation/beet": "0.0.0",
+    "@metaplex-foundation/beet-solana": "0.0.0",
+    "@solana/web3.js": "^1.31.0"
+  },
+  "devDependencies": {
+    "eslint": "^8.3.0",
+    "prettier": "^2.5.1",
```

Comment:
WRT those (including eslint) they are also devDeps in the root package.json and in order to keep versions of those compatible we should probably not include them per package.

Reply:
OK weird this worked before ... maybe it's due to switching yarn version?

For now let's keep this and figure out later.